app/services/ReturnsService.scala

```diff
@@ -164,6 +164,16 @@ class ReturnsService @Inject()(val vatRegConnector: VatRegistrationConnector,
     }
   }
 
+  def saveNorthernIrelandReceiveGoods(receiveGoods: Boolean, amount: Option[BigDecimal])
```

Comment:
No need for this function as this can be handled in the controller using already existing functions like with the latest overseasCompliance pages we've added